src/core/scenario.jl

```diff
@@ -400,3 +458,57 @@ function add_doe_reference_names_from_elec_to_thermal_loads(elec::Dict, thermal:
         end
     end
 end
+
+"""
+    function set_missing_emissions_factors_to_fuel_defaults(tech::AbstractFuelBurningTech)
+
+Populate emission factors in tech based on its fuel_type field.
+"""
+function set_missing_emissions_factors_to_fuel_defaults(tech::AbstractFuelBurningTech)
```

Comment:
Can we use this function in the tech constructors so that we do not have to make the techs mutable?